lib/Dialect/TTNN/Transforms/TTNNLayout.cpp

```diff
@@ -263,6 +263,13 @@ static bool changeLayoutToHost(DestinationStyleOpInterface &op,
   return false;
 }
 
+bool shouldOpHaveTileResultDefault(Operation *op) {
```

Comment:
@sdjordjevicTT, is a more suitable place for this?  Arguably this should use the WA infra you added which is capable of tagging constraints on tensor layouts, though I think it might only currently apply to operands?  In this case we need a mechanism for encoding that the result is row major.

Reply:
@sdjordjevicTT has actually informed me earlier this evening that his pr here allows for a solution to this problem. Hes also added the workaround I need:

https://github.com/tenstorrent/tt-mlir/pull/1613

Reply:
Awesome, I suppose we can close this PR then?

Reply:
Sounds good.